lib/embulk/output/google_spreadsheets.rb

```diff
@@ -134,16 +166,29 @@ def self.clean_previous_records(worksheet, schema, task)
         row_range    = task["row_index"]..worksheet.num_rows
         column_range = task["col_index"]...(task["col_index"] + schema.length)
 
-        row_range.each do |row|
-          column_range.each do |col|
-            worksheet[row, col] = ''
+        row_range.each do |r|
+          column_range.each do |c|
+            worksheet[r, c] = ''
           end
         end
       end
 
       def format(type, v)
-        return @null_representation if v.nil?
-        v
+        return @null_string if v.nil?
+
+        case type
+        when :timestamp
+          zone_offset = task['default_timezone']
+          zone_offset = '+00:00' if zone_offset == 'UTC'
```

Comment:
We should do such conversion only one time I this plugin rather than doing for each value.

How about making an instance method which memoizes the converted value with an instance variable.